project-p1-AnaB/code/drawings.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Nora's Geschichten</title>
+  <link rel="stylesheet" type="text/css" href="style.css" />
+  <link rel="icon" type="image/x-icon" href="./Images/favicon.ico" />
+</head>
+
+
+<body>
+  <header>
+    <div class="title_with_image">
+      <h1>Nora's Kunstecke</h1>
+      <img src="./Images/Rainbow.png" alt="Regenbogen" class="icon">
+    </div>
+    <nav>
+      <a href="index.html" class="yellow">
+        <img src="./Images/PaintBrush.png" alt="Zur Kunstecke" class="img_nav" />
+        <h2>Zur Kunstecke</h2>
+      </a>
+      <a href="stories.html" class="green">
+        <img src="./Images/BookStack.png" alt="Lies eine Geschichte" class="img_nav" />
+        <h2>Geschichten</h2>
+      </a>
+      <a href="contact.html" class="red">
+        <img src="./Images/BookStack.png" alt="Schreib mir" class="img_nav" />
+        <h2>Kontakt</h2>
+      </a>
+    </nav>
+  </header>
+
+  <section class="image_gallery">
+    <img id="drawings" src="./Images/Drawing Ed Edda.jpg" alt="Ed und Edda"/>
+    <img id="drawings" src="./Images/Drawing house.jpg" alt="Haus mit Perspektive"/>
+    <img id="drawings" src="./Images/Drawing Marin.jpg" alt="Marin"/>
+    <img id="drawings" src="./Images/Drawing Marin 2.jpg" alt="Marin"/>
+    <img id="drawings" src="./Images/Drawing Picachu.jpg" alt="Picachu"/>
+    <img id="drawings" src="./Images/Drawing Pingi.jpg" alt="Pingi"/>
+    <img id="drawings" src="./Images/Drawing_Nora.jpg" alt="Nora"/>
+    <img id="drawings" src="./Images/Lloyd.jpg" alt="Lloyd" />
+
+    <div id="modal_drawings" class="modal">
+      <span class="closemodal">&times;</span>
+      <img class="drawingzoom" id="img01">
+      <div id="caption"></div>
+    </div>
+  </section>
+
+  <script>
+    var modal = document.getElementById("modal_drawings");
+    var img =document.getElementById("drawings");
+    var modalImg = document.getElementById("img01");
+    var captionText = document.getElementById("caption");
+
+    img.onclick = function() {
+      modal.style.display = "block";
+      modalImg.src = this.src;
+      captionText.innerHTML = this.alt;
+    }
+
+    var span = document.getElementsByClassName("closemodal")[0];
+
+    span.onclick = function () {
+      modal.style.display = "none";
+    }
+
+  </script>
```

Comment:
Very nice feature! Just be aware that it only works for the first image. The reason behind this is that the attribute id, is unique - only one element can have it.

Reply:
@HIPPIEKICK Thank you for your comments! yes, the repo is a mess, but not sure how to remove unneeded files :/
regarding the js comment, i realized that yes, any suggestion how to solve it? :)

Thanks again for taking time